src/app/shared/models/person.ts

```diff
@@ -64,23 +59,32 @@ export class Person {
     return this.devices.includes(device);
   }
 
-  hasSupervisorRating(): boolean {
-    return this.supervisorRating !== undefined && this.supervisorRating !== SkillLevel.None;
+  hasSupervisorAssessment(): boolean {
+    return this.supervisorAssessment !== undefined && this.supervisorAssessment !== SkillLevel.None;
   }
 
   getiOSSkillLevel(): SkillLevel {
-    const iOSSkillLevel = Person.IOSDevExperienceAnswerToSkillLevelMap.get(this.iosDev);
-    return iOSSkillLevel !== undefined ? iOSSkillLevel : SkillLevel.None;
+//     const iOSSkillLevel = Person.IOSDevExperienceAnswerToSkillLevelMap.get(this.iosDev);
+//     return iOSSkillLevel !== undefined ? iOSSkillLevel : SkillLevel.None;
+
+    // returning placeholder value while transforming model
+    // iOS skill level should be retrieved from the iOS skill in the skills array
+    return SkillLevel.Low;
   }
 
-  getIntroAssessmentLevel(): SkillLevel {
-    const index = CSVConstants.Person.IntroAssessmentAnswers.indexOf(this.introAssessment);
+  getIntroSelfAssessmentLevel(): SkillLevel {
+    const index = CSVConstants.Person.IntroSelfAssessmentAnswers.indexOf(this.introSelfAssessment);
     return index >= 0 ? index : null;
   }
 
-  getIntroAssessmentTutorLevel(): SkillLevel {
-    const index = CSVConstants.Person.IntroAssessmentTutorAnswers.indexOf(this.introAssessmentTutor);
-    return index >= 0 ? index : null;
+  getSupervisorAssessmentLevel(): SkillLevel {
+//     const index = CSVConstants.Person.SupervisorRating.indexOf(this.supervisorAssessment);
+//     return index >= 0 ? index : null;
+
+    // returning placeholder value while model transformation is underway
+    // we no longer store the intro course assessment by tutors separately from the supervisor assessment
+    // therefore this no longer needs to use 'IntroAssessmentTutorAnswers'
+    return SkillLevel.Low;
```

Comment:
Why did you hard-code theses skill levels here?

Reply:
[Fixed](https://github.com/ls1intum/team-allocator/pull/43/commits/199f7392ae822a611f97225db53d116c516eac1c), individual skill levels should be correctly parsed again now too so that not everyone is unrated.

Reply:
Should be fixed in [ea74e3b](https://github.com/ls1intum/team-allocator/pull/43/commits/ea74e3b6b440a1e486f8564c1a3881064bbfb12e), iOS skill level is now retrieved from the iOS skill in the skills array.